### PR TITLE
update CI for current deployment targets

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -5,21 +5,17 @@ set -e
 export LC_ALL=en_US.UTF-8
 
 TEST_ON_EL=$([ -f /etc/redhat-release ] && [ -x /usr/bin/yum ] && echo "true" || echo "false")
```
Comment (ekohl) [inline test.sh]: Should this also become 
```suggestion
TEST_ON_EL=$([ -f /etc/redhat-release ] && [ -x /usr/bin/dnf ] && echo "true" || echo "false")
```

Comment (evgeni) [reply test.sh]: Maybe? ;-)

I honestly do not remember what I thought back then in https://github.com/Katello/katello-certs-tools/commit/1398fa7c637f34454af66cb9939e23e20df593c9

Shouldn't `/etc/redhat-release` be sufficient?

Comment (ekohl) [reply test.sh]: I think so too. Perhaps yum wasn't available on very bare containers?

Comment (evgeni) [reply test.sh]: but then the tests wouldn't run anyway :shrug:

**Review by ehelms — APPROVED** [review]

**Pull request merged**